deployment/ccip/changeset/test_environment.go

```diff
@@ -441,19 +453,50 @@ func NewEnvironmentWithJobsAndContracts(t *testing.T, tc *TestConfigs, tEnv Test
 			Changeset: commonchangeset.WrapChangeSet(commonchangeset.DeployMCMSWithTimelock),
 			Config:    mcmsCfg,
 		},
+	})
+	require.NoError(t, err)
+	tEnv.UpdateDeployedEnvironment(e)
+	e = AddCCIPContractsToEnvironment(t, e.Env.AllChainSelectors(), tEnv)
+	// now we update RMNProxy to point to RMNRemote
+	e.Env, err = commonchangeset.ApplyChangesets(t, e.Env, nil, []commonchangeset.ChangesetApplication{
```

Comment:
Noticed that this isn't in `AddCCIPContractsToEnvironment`, is that intentional? This caught me when writing the add chain test

Reply:
Yeah. RMNProxy can be owned by timelock like in this test case. If owned by timelock, we will have to run this changeset with MCMS config but that's not the case for generic test set up. that's why I have kept it separate.